modules/container-service/managed-cluster/main.bicep

```diff
@@ -405,18 +459,14 @@ resource managedCluster 'Microsoft.ContainerService/managedClusters@2023-05-02-p
     tier: aksClusterSkuTier
   }
   properties: {
+    httpProxyConfig: !empty(httpProxyConfig) ? httpProxyConfig : null
+    identityProfile: !empty(identityProfile) ? identityProfile : null
     diskEncryptionSetID: !empty(diskEncryptionSetID) ? diskEncryptionSetID : null
     kubernetesVersion: (empty(aksClusterKubernetesVersion) ? null : aksClusterKubernetesVersion)
     dnsPrefix: aksClusterDnsPrefix
     agentPoolProfiles: primaryAgentPoolProfile
     linuxProfile: (empty(aksClusterSshPublicKey) ? null : aksClusterLinuxProfile)
     servicePrincipalProfile: (empty(aksServicePrincipalProfile) ? null : aksServicePrincipalProfile)
-    ingressProfile: {
```

Comment:
Why was this deleted? 

I see we still have the `dnsZoneResourceId` input parameter and a role assignment on the bottom of the template. Yet the cluster doesn't know about the DNS Zone anymore :o

Reply:
glitch in the matrix :D restored.